app/uk/gov/nationalarchives/omega/editorial/views/editSet.scala.html

```diff
@@ -3,9 +3,8 @@
 @(title: String, heading: String, editSet: EditSet)
 
 @template(title) {
-  <h1>@heading</h1>
-
-  <table class="govuk-table">
+ <table class="govuk-table">
+    <caption class="govuk-table__caption govuk-table__caption--l">@heading</caption>
```

Comment:
Can we use `govuk-table__caption--m` please as per the prototype: https://ctd-omega-ui-prototype.herokuapp.com/edit-set